src/components/Editor/Editor.css

```diff
@@ -126,6 +126,10 @@ html[dir="rtl"] .editor-mount {
   height: 100%;
 }
 
+.CodeMirror-scroll {
+  margin-right: 0px;
+}
```

Comment:
why is this needed?

Reply:
When I was using the codeMirror, I noticed that the clientWidth that was being returned from the getScrollInfo() was not matching what was seen in the inspector. It was off by 30px. I found that by default it had a margin-right: -30px. Although once I removed this extra negative margin, it properly represented the scroll area, the bounds of the scroll view.

Reply:
@jasonLaster @mmcote you might find this discussion interesting: https://github.com/codemirror/CodeMirror/issues/2984
not the same margin, but maybe the same reason.

Reply:
Thank you for bringing this to my attention :) I will make some corresponding changes when I get a chance later on in the day.